src/languages/en/ui.js

```diff
@@ -48,4 +48,5 @@ export const ACTIVE_TEAMS = 'Active Teams';
 export const ADMIN = 'Admin';
 export const POPUP_MANAGEMENT = 'Popup Management';
 export const CANCEL = 'Delete Final Day';
-export const SET_FINAL_DAY = 'Set Final Day';
\ No newline at end of file
+export const SET_FINAL_DAY = 'Set Final Day';
+export const Management = 'Manage Final Day';
```

Comment:
It's better to use all caps here for consistency. Change it to `export const MANAGE_FINAL_DAY = 'Manage Final Day'`. Don't forget change all the places it is used.